CRM/Api4/Services.php

```diff
@@ -94,9 +99,16 @@ public static function loadServices($namespace, $tag, $container) {
             $definition->addTag($tag);
           }
         }
+        $container->addResource($resource);
       }
       catch (\InvalidArgumentException $e) {
         //Directory is not found so lets not do anything i suppose.
+
+        // FIXME: The above comment implies that the try/catch is specifically
+        // about the directory's existence, which would make it redundant with the
+        // newer "file_exists()" guard. However, the actual "catch()" seems broader,
+        // and I don't see anything in DirectoryResource that throws the exception.
+        // So... maybe it's not needed, or maybe there's some additional purpose.
```

Comment:
@totten to address this question, the above try/catch was added in #15765 specifically to address the directory not existing. That was its only purpose so if it's now redundant I would say yes this can be removed.

Reply:
@colemanw Cool. I've cleaned up that bit and re-tested.